school-library/teachers/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require '../person/persons'
+
+class Teachers < Person
+  def initialize(specialization)
```

Comment:
- [ ] The constructor parameters of the `Teacher` class should also include all parameters previously in the `Person` class. .e.g. `def initialize(age, specialization, name = "Unknown", parent_permission = true)`. Kindly fix this